src/components/GlobalLearningResourcesPage/GlobalLearningResourcesFilters.tsx

```diff
@@ -37,13 +37,18 @@ const GlobalLearningResourcesFilters: React.FC<
     _event: React.FormEvent<HTMLInputElement>,
     value: string
   ) => {
-    const trimmedValue = value.trim();
-    if (trimmedValue) {
-      setLoaderOptions({
-        ...(loaderOptions || loaderOptionsFalllback),
-        'display-name': trimmedValue,
-      });
+    const lastChar = value[value.length - 1];
```

Comment:
What if we did this check ibn the fetch method? The issue here is that we use the loader options both as an API param and react state variable. While the react state variable can in fact have any value, the API param might not. If we start to mess around with the value before the state variable is set, we will always have issues or inconsistencies.